### ISSUE TITLE
Missing track information

### ISSUE THREAD

**Issue by jarisgv** [nulloy/nulloy#146]
![nulloy-missing-info-1](https://user-images.githubusercontent.com/19142770/51226047-bd834000-1934-11e9-8ccf-b08cbe811a91.png)

Hi guys !
I just started using Nulloy 0.8.2 - Portable.
I realized that the player isn't displaying the track information for some mp3 files.
All the files are tagged properly with title, artist, album artist, genre etc. 
The only thing I know is that it doesn't occur because of the length of the files or because the directory has many songs.
Wait for your reply.
Greetings from Brazil :brazil:

**Comment by noscript** [comment]
Hi! Would you please try the pre-release build? http://nulloy.com/download/prerelease/

**Comment by jarisgv** [comment]
Thnx @noscript !
I'll check it out.

**Comment by jarisgv** [comment]
Unfortunately doesn't work.
![image](https://user-images.githubusercontent.com/19142770/51267223-64eb8b80-19a4-11e9-80e6-8d25145e8374.png)

**Comment by jarisgv** [comment]
Mmm. So strange. I deleted the stable version and it started to work fine.
Is is using any settings outside of the portable directory?
I see that the app generates a temp file called "**qtsingleapp-nulloy-81cf-1-lockfile**" in **"C:\Users\...\AppData\Local\Temp"**, but this file is blank.
![image](https://user-images.githubusercontent.com/19142770/51267692-77b29000-19a5-11e9-8295-5152837e952f.png)

**Comment by noscript** [comment]
> Is is using any settings outside of the portable directory?

No, it will store the settings in the local directory. However, if you extract the archive under `C:\Program Files ...` then it will store the setting in `C:\Users\...\AppData\Roaming\Nulloy`

>I see that the app generates a temp file called "qtsingleapp-nulloy-81cf-1-lockfile" in "C:\Users...\AppData\Local\Temp", but this file is blank.

`qtsingleapp-nulloy-...-lockfile` is a dummy file used to prevent running multiple instances of the player. That file will be deleted once the player exits.

>It seems like the error occurs just because I set the arrow 'Up' to 'Play previous track in playlist' and arrow 'Down' to 'Play the next track in playlist' as hotkeys in Keyboard settings. The error occurs even in 'Shortcuts' and 'Global Shortcuts' options.
When I reset it to 'Z' and 'B' as the default keys, the track information is displayed again - BUT - just in the pre-release version.

I managed to reproduce it. For some reason the track information is not loaded for the file in playlist, unless playback is restarted.


P.S. Pre-release builds are always the newers.

**Comment by jarisgv** [comment]
Thanks for replying @noscript.
But everything looks fine here. 
I just need to get used to using the 'Z' and 'B' keys to skip the song tracks.
:smile:
